InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -27,7 +27,7 @@
 from health_ml.utils import log_on_epoch 
 
 RESULTS_COLS = [ResultsKey.SLIDE_ID, ResultsKey.TILE_ID, ResultsKey.IMAGE_PATH, ResultsKey.PROB,
-                ResultsKey.PROB_CLASS, ResultsKey.PRED_LABEL, ResultsKey.TRUE_LABEL, ResultsKey.BAG_ATTN]
+                ResultsKey.CLASS_PROB, ResultsKey.PRED_LABEL, ResultsKey.TRUE_LABEL, ResultsKey.BAG_ATTN]
```

Comment:
I'd be more in favour of `CLASS_PROBS` (plural), because it will always refer to all classes, but I leave it up to you.

Reply:
Well-spotted, thanks. Changed now